estimation.py

```diff
@@ -184,13 +183,14 @@ def loss_fn(pred, labels):
                     torch.nn.utils.clip_grad_norm_(
                         model.parameters(), job_config.training.max_norm, foreach=True
                     )
+                # sync float8 amaxes and scales
+                float8_handler.sync_float8_amax_and_scale_history(model)
```

Comment:
this is probably because train.py has it but estimation.py is outdated?

Reply:
I think so. @sanketpurandare to double check.